templates/internet-of-things/digital-signage.html

```diff
@@ -201,14 +201,13 @@ <h3 class="p-card__title">4YouSee</h3>
     <div class="col-6 p-card">
       <header class="no-border u-align--center u-vertically-center u-hide--small" style="min-height: 7rem;">
         <a href="https://www.intel.co.uk/content/www/uk/en/products/boards-kits/nuc.html">
-          {{
-            image(
-              url="https://assets.ubuntu.com/v1/c24cb4b9-logo-intel.svg",
-              alt="Intel® NUC",
-              height="65",
-              width="98",
-              hi_def=True,
-              loading="lazy"
+          {{ image (
```

Comment:
Could you give it the same format as before with the `image` word in a new line?